network/server/tchannelthrift/node/service.go

```diff
@@ -632,6 +632,47 @@ func (s *service) SetWriteNewSeriesAsync(
 	return s.GetWriteNewSeriesAsync(ctx)
 }
 
+func (s *service) GetMaxWiredBlocks(
+	ctx thrift.Context,
+) (*rpc.NodeMaxWiredBlocksResult_, error) {
+	runtimeOptsMgr := s.db.Options().RuntimeOptionsManager()
+	value := runtimeOptsMgr.Get().MaxWiredBlocks()
+	return &rpc.NodeMaxWiredBlocksResult_{
+		MaxWiredBlocks: int64(value),
+	}, nil
+}
+
+func (s *service) SetMaxWiredBlocks(
+	ctx thrift.Context,
+	req *rpc.NodeSetMaxWiredBlocksRequest,
+) (*rpc.NodeMaxWiredBlocksResult_, error) {
+	runtimeOptsMgr := s.db.Options().RuntimeOptionsManager()
+	set := runtimeOptsMgr.Get().SetMaxWiredBlocks(int(req.MaxWiredBlocks))
+	runtimeOptsMgr.Update(set)
+	return s.GetMaxWiredBlocks(ctx)
+}
+
+func (s *service) GetWiredBlockExpiryAfterNotAccessedPeriod(
+	ctx thrift.Context,
+) (*rpc.NodeWiredBlockExpiryAfterNotAccessedPeriodResult_, error) {
+	runtimeOptsMgr := s.db.Options().RuntimeOptionsManager()
+	value := runtimeOptsMgr.Get().WiredBlockExpiryAfterNotAccessedPeriod()
+	return &rpc.NodeWiredBlockExpiryAfterNotAccessedPeriodResult_{
+		ExpiryMilliseconds: int64(value / time.Millisecond),
```

Comment:
Well other Thrift APIs actually use `timestamp` and `timestampType`.  I can change this to `duration` and `durationType` and allow SECONDS, MILLISECONDS, MICROSECONDS, NANOSECONDS to be used.